client/post-editor/post-editor.jsx

```diff
@@ -213,6 +215,18 @@ export const PostEditor = React.createClass( {
 		}
 	},
 
+	getSelectedText: function() {
```

Comment:
`get*()` method names traditionally refer to a method that returns a value. This would probably be better named `copySelectedText()`, `storeSelectedText()`, or something of that nature.